src/index.js

```diff
@@ -20,12 +20,13 @@ const useScreenshot = ({ type, quality } = {}) => {
   /**
    * convert html node to image
    * @param {HTMLElement} node
+   * @param {Options} html2canvas options
    */
-  const takeScreenShot = (node) => {
+  const takeScreenShot = (node, options = {}) => {
```

Comment:
Great idea, this is gonna fix a lot of peoples' problems.

nit: no need for ` = {}`

Reply:
@anthonykrivonos backward compatibility?

Reply:
I think, there is a bug here. When you take screenshot, it has not image. Ony ıt has text and bacground.

Reply:
@MustafaTRKYLMZ
did you give the option like `useCORS: true` as described in the example above??